frontend/institution/manageInstMenu/manageInstMenuController.js

```diff
@@ -0,0 +1,66 @@
+"use strict";
+
+(function() {
+    angular
+    .module("app")
+    .controller("ManageInstMenuController", [
+        'AuthService', 'ManageInstItemsFactory',
+        ManageInstMenuController]);
+
+    function ManageInstMenuController(AuthService, ManageInstItemsFactory) {
+        const manageInstMenuCtrl = this;
+
+        manageInstMenuCtrl.$onInit = () => {
+            _.defaults(manageInstMenuCtrl, {
+                user: AuthService.getCurrentUser(),
+            });
+            
+            manageInstMenuCtrl._loadSwitchInstOptions();
+            manageInstMenuCtrl._loadInstitution();
+        };
+
+        manageInstMenuCtrl._loadInstitution = () => {
```

Comment:
is it really necessary? Why don't u set manageInstMenuCtrl.institution = user.current_institution?